### ISSUE TITLE
Support killing forked runs

### ISSUE THREAD

**Issue by jedesah** [sbt/sbt#3948]
## steps

enable `fork in run := true`
run `run`

## problem

It would be really convenient if at this point there was an easy way to kill the forked process without having to use `kill` in another terminal window after having found the PID of the new process

## expectation

Perhaps something such as `ctrl-d` (I think the play framework does this) would be a nice way to expose this functionality.

**Comment by dwijnand** [comment]
Doesn't ctrl-c kill the forked process if fork is enabled?

**Comment by jedesah** [comment]
No, it kills the `sbt` console. Unless something odd is going on with my particular project. I'll do some tests with a vanilla project and see

**Comment by chorandrey** [comment]
@jedesah might [this plugin](https://github.com/spray/sbt-revolver) be solution/convenient workaround for you?

**Comment by jedesah** [comment]
@chorandrey Yes, I think that would work! Thx for pointing it out! Now that I think of it, I think I've used it in the past but I forgot the name of it which is what made so frustrating to not be able to get back to that same workflow! Thx again!

I still think it would be reasonable for this to be natively supported so I'll leave this open and let a maintainer close it if they just don't want to ever support this in sbt itself

**Comment by eed3si9n** [comment]
```
sbt:hello> run
[info] Compiling 1 Scala source to /private/tmp/foo/target/scala-2.13/classes ...
[info] running (fork) example.Hello
[info] hello
[info] hello
[info] hello

// Ctrl-C

[warn] Canceling execution...
[error] Total time: 6 s, completed Oct 1, 2019 4:50:29 PM
```

This was fixed in 1.3.0.
